joomla/joomla-cms
# [4.0] com_finder strings

### Summary of Changes
Update code so we can have more specific strings such as Maps disabled and Filters disabled instead of the generic Item disabled

### Testing Instructions
Make sure you have indexed the content and then check the messages when publishing/unpublishing etc
